services/static-webserver/client/source/class/osparc/utils/Avatar.js

```diff
@@ -257,8 +260,9 @@ qx.Class.define("osparc.utils.Avatar", {
         return i.toLowerCase();
       };
 
-      return "https://secure.gravatar.com/avatar/" + MD5(email) + "?s=" + size + "&d=" + defIcon + "&r=" + rating;
-    }
-
+      const emailHash = MD5(email);
+      const defaultImageUrl = `https://ui-avatars.com/api/${username}/${size}`;
```

Comment:
Q: this code somewhere handles the url encoding, right? because `defaultImageUrl` goes in the query